nubis/puppet/consul.pp

```diff
@@ -69,6 +69,13 @@
       verbose => true,
       redownload => true, # The file already exists, we replace it
```

Comment:
Indeed it does! I looked at the module briefly and missed the `$mode` parameter. I'll get change the PR to reflect that